app/controllers/users_controller.rb

```diff
@@ -11,10 +11,27 @@ def create
   end
 
   def show
-    @user = User.find(session[:user_id])
+    @user = current_user
```

Comment:
Nice use of the helper method current_user here! I think I can also utilize this in my user stories.

Reply:
So basically, in the user#show method, if we leave the @user =User.find(session[:user_id]), it cannot find the session[:user_id], since we never actually login in to start that session. So by setting @user equal to current_user instead, we can just stub any instance of 'current_user' from the application controller with our test object @regular_users... so we don't have to rely on a session[:user_id] find the user in the db... https://stackoverflow.com/questions/7448974/how-to-stub-applicationcontroller-method-in-request-spec